src/components/select/index.tsx

```diff
@@ -31,6 +31,7 @@ export const Select = forwardRef((props: SelectProps, ref: ForwardedRef<SelectRe
         disabled = false,
         flatListProps,
         hideSelectControlArrow,
+        isAnimated = true,
```

Comment:
The default value for this props should be `false`. Setting it as `true` sounds like a [breaking change](https://semver.org/lang/pl/), so we would have to release 2.0.0, but we don't want to do (at least for now 😄 ). The next version should be 1.2.0 and the user should be able to enable it if want to. In the future, in version 2.0.0, I am definitely for set animations in default :)